azext_hanaonazure/azext_metadata.json

```diff
@@ -1,4 +1,4 @@
 {
   "azext.minCliCoreVersion": "2.0.46",
-  "version": "0.3.9"
+  "version": "0.4.0"
```

Comment:
nit: i would do 3.10. going to 4.0 implies big changes. maybe when we ship, we can change it to 4.0

Reply:
+1 I like this idea; test to see if this works ok. https://microsoft.sharepoint.com/teams/AzureDedicatedEngineeringTeam/_layouts/OneNote.aspx?id=%2Fteams%2FAzureDedicatedEngineeringTeam%2FSiteAssets%2FAzure%20Dedicated%20Engineering%20Team%20Notebook&wd=target%28SAP%20HANA.one%7CFB7D0654-1276-104F-AF53-BA55FC4DA39E%2FAzure%20CLI%7C7303BF06-991D-4331-9BF8-542F65E5D86E%2F%29

Reply:
done and tested locally